run_filters.py

```diff
@@ -0,0 +1,168 @@
+
+        print >> sys.stderr, "--------------Replacing drug IDs--------------------"
+        input_path = os.path.join(folder_name, "clean.csv")
+        output_path = os.path.join(folder_name, "drugs.csv")
+        with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+            filter_replace_drug_id(input_ptr, filter_meta, output_ptr)
+
+        print >> sys.stderr, "--------------Fixing C1S in files--------------------"
+        input_path = os.path.join(folder_name, "drugs.csv")
+        output_path = os.path.join(folder_name, "c1s.csv")
+        with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+            filter_fix_c1s(input_ptr, filter_meta, output_ptr)
+
+            # get_replaced_file(input_ptr,output_ptr)
+
+        # print >> sys.stderr, "--------------Fixing FVP in files--------------------"
+        # input_path = os.path.join(folder_name, "c1s.csv")
+        # output_path = os.path.join(folder_name, "fixed_fvp.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_fix_fvpheader(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Filling in Defaults--------------------"
+        # input_path = os.path.join(folder_name, "fixed_fvp.csv")
+        # output_path = os.path.join(folder_name, "default.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_fill_default(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Updating fields--------------------"
+        # input_path = os.path.join(folder_name, "default.csv")
+        # output_path = os.path.join(folder_name, "Update_field.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_update_field(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Removing Unnecessary Records--------------------"
+        # input_path = os.path.join(folder_name, "Update_field.csv")
+        # output_path = os.path.join(folder_name, "CleanedPtid_Update.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_remove_ptid(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Removing Records without VisitDate--------------------"
+        # input_path = os.path.join(folder_name, "CleanedPtid_Update.csv")
+        # output_path = os.path.join(folder_name, "final_Update.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_eliminate_empty_date(input_ptr, filter_meta, output_ptr)
+
+    except Exception as e:
+        print "Error in Opening a file"
+        print e
+
+
+    return
+
+def connect_to_redcap(config_path):
```

Comment:
This name doesn't correctly identify what the function is doing. Maybe it should be called something like get_config or read_config, something a little more identifying to the function.